biojava-core/src/main/java/org/biojava/nbio/core/sequence/io/embl/EmblId.java

```diff
@@ -0,0 +1,108 @@
+package org.biojava.nbio.core.sequence.io.embl;
+
+/**
+ *
+ * This class contains the processed data of embl file
+ * Primary accession number
+ * Sequence version number
+ * Topology: 'circular' or 'linear'
+ * Molecule type
+ * Data class
+ * Taxonomic division
+ * Sequence length
+ * @since 5.0.0
+ * @author Noor Aldeen Al Mbaidin
+ */
+public class EmblId {
+
+
+    private String primaryAccession;
+    private String sequenceVersion;
+    private String topology;
+    private String moleculeType;
+    private String dataClass;
+    private String taxonomicDivision;
+    private String sequenceLength;
+
+    public EmblId() {
```

Comment:
This no-arg public constructor can be removed

Reply:
done.